torchvision/prototype/transforms/functional/__init__.py

```diff
@@ -172,3 +172,4 @@
 )
 
 from ._deprecated import get_image_size, rgb_to_grayscale, to_grayscale, to_tensor  # usort: skip
+from ._temporal import uniform_temporal_subsample_video, uniform_temporal_subsample  # usort: skip
```

Comment:
Why did you put it at the end of the file? Imports from `._deprecated` are at the end since they potentially need to import from the other namespaces and thus should be the last module to load. This is not true for `_temporal`, right? If so, please remove the `# usort: skip` directive and put it in the large block above.